src/styles/index.js

```diff
@@ -1 +1,2 @@
 require('./ux-aspects.less');
+require('./hpe-icons.css');
```

Comment:
This was supposed to be optional, why bundle it back in here?

Reply:
@donna-marijne It was added here for the plunker examples to be able to use it. I'll speak to Ashley to try and work out a way to do it differently.

Reply:
Okay, if it's something specific to the documentation then it should be under the docs directory.

Reply:
Like `docs/styles` instead of `src/styles`. `src` is the library content.

Reply:
Sure that makes more sense. I'll move it now.